### PR TITLE
feat #102199: Satellite images base

### PR DESCRIPTION
**Pull request by ErlendAHall**
## Checklist
- [x] I have run linter. `npm run lintfix`
- [x] I have covered potential overrides that will happen in [reboot.scss.](https://github.com/twbs/bootstrap/blob/main/scss/_reboot.scss)
- [ ] I added some tests. (to be defined further) [optional]
- [ ] I have updated [the wiki.](https://github.com/equinor/echo-maps/wiki) [optional]

## What's new
Added the base setup for satellite images layer. No UI except placeholders and markers are present here.

### CODE REVIEW

File: src/enums/LayerIds.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export enum LayerIds {
     SUBSEA_WORK_ORDER = 'subsea-work-order',
     MARINE_INFRASTRUCTURE = 'marine-infra',
     WELL_BARRIER_OVERVIEW = 'well-barrier-overview',
+    SATELITTE_IMAGES = 'satellite-images',
```
Comment (olsoybakk) [inline src/enums/LayerIds.ts:12]: SATELLITE_IMAGES

File: src/layers/external/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export * from './leakage';
 export * from './timp';
 export * from './wbo';
+export * from './satelitte-images';
```
Comment (olsoybakk) [inline src/layers/external/index.ts:4]: rename folder to _satellite-images_

File: src/enums/LayerIds.ts (hunk 1 above)
Comment (ErlendAHall) [reply src/enums/LayerIds.ts]: good eagle eyes

**Review by olsoybakk — APPROVED** [review]
NickCage is back 😂

**Pull request merged**